src/util/pagination/paged-result.js

```diff
@@ -1,23 +1,24 @@
-let config = require('../../config/config')()
 const TOTAL_COUNT_FIELD = 0
 const DATA_FIELD = 1
 
 /**
  * @pagedResult: It is required that the result parameter contains the total count
  * of records at index 0 and the array of items at index 1.
  */
-module.exports = (req, page = 0, pageSize = parseInt(config.PAGE_SIZE), data) => {
+module.exports = (req, data) => {
   const REGEX = /(\?(page)=(\d+))/gi
-  let host = `${req.protocol}://${req.headers.host}`
-  let url = `${req.originalUrl.replace(REGEX, '')}`
+  const host = `${req.protocol}://${req.headers.host}`
+  const url = `${req.originalUrl.replace(REGEX, '')}`
 
-  let totalPages = parseInt((data[TOTAL_COUNT_FIELD] / pageSize).toFixed())
+  const totalPages = parseInt((data[TOTAL_COUNT_FIELD] / req.query.pageSize).toFixed())
```

Comment:
it will not reach this code without the proper parameters. That's the [page validation middleware](https://github.com/Ivan-Marquez/momentum/blob/hotfix/refactor/src/middleware/page-validations.js) main purpose.